Automattic/vip-go-mu-plugins
# Standardize two factor authentication SMS messages

## Description

Standardizing the message format for SMS messages used for two factor authentication to help enhance security practices.

Read more about the standardization proposal by Webkit here: https://github.com/WebKit/explainers/tree/master/sms-one-time-code-format

Old format:
`The verification code you requested is: 123456`

New format:
```
123456 is your FooBar verification code.

@foobar.com #123456
```

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Go to `wp-admin` > `Edit My Profile` > `Two-Factor Options` > `SMS`
3. Enter your SMS number and click Submit
4. Check your phone and verify the message format. It should look like:
```
123456 is your FooBar verification code.

@foobar.com #123456
```
5. PHPunit tests
Run `./bin/phpunit-docker.sh tests/wpcom-vip-two-factor/sms-provider.php`

This has been tested on a Go Sandbox using a test site 👍
